archive/dto.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: dto
+  title: DTB/DTBO
+  license: CC0-1.0
+  file-extension: img
+  endian: be
+
```

Comment:
Would you add at least a few sentences about the format in [the `/doc` key](https://doc.kaitai.io/ksy_style_guide.html#doc), i.e. some general information about the format? What it is used for? What kind of data it stores? What is the main usage field?

The point is for someone who visits the format page in our format gallery to get a basic idea about what kind of the format it is, and not have to search it on Wikipedia or other external websites.